CoreWordCloudView/src/au/gov/asd/tac/constellation/views/wordcloud/ui/WordCloudIOProvider.java

```diff
@@ -79,12 +79,8 @@ public void readObject(final int attributeId, final int elementId, final JsonNod
                 final int setSize = iter.next().asInt();
                 final Set<Integer> wordSet = new HashSet<>();
                 for (int i = 0; i < setSize; i++) {
-                    int elID = iter.next().asInt();
-                    if (elementType == GraphElementType.VERTEX) {
-                        elID = vertexMap.get(elID);
-                    } else {
-                        elID = transactionMap.get(elID);
-                    }
+                    int elID = iter.next().asInt();                        
+                    elID = (elementType == GraphElementType.VERTEX) ? vertexMap.get(elID): transactionMap.get(elID);
```

Comment:
just looking at this. It seems like these 2 lines should be editing 2 separate variables as they appear to refer to 2 separate things (first being iterating through iter, 2nd using iter value to extract from map)